10-inference.Rmd

```diff
@@ -83,124 +87,120 @@ The population consists of all studio apartment rentals in Vancouver, and the po
-set.seed(1)
-samples_1 <- rep_sample_n(tbl = virtual_box, size = 40)
-choc_sample_1 <- summarize(samples_1,
-  n = sum(flavour == "chocolate"),
-  prop = sum(flavour == "chocolate") / 40
+set.seed(123)
+sample_1 <- rep_sample_n(tbl = airbnb, size = 40)
+airbnb_sample_1 <- summarize(sample_1,
+  n = sum(room_type == "Entire home/apt"),
+  prop = sum(room_type == "Entire home/apt") / 40
 )
-choc_sample_1
+airbnb_sample_1
 ```
-Here we see that the proportion of chocolate Timbits in this random sample is
-`r round(choc_sample_1$prop,2)`. This value is our estimate &mdash; our best guess of our population parameter using this sample. Given that it is a single
+Here we see that the proportion of entire home/apartment listings in this random sample is
+`r round(airbnb_sample_1$prop,2)`. This value is our *estimate* &mdash; our best guess of our population parameter using this sample. Given that it is a single
 value that we are estimating, we often refer to it as a **point estimate**.
 
-Now imagine we took another random sample of 40 Timbits from the population. Do you
+Now imagine we took another random sample of 40 listings from the population. Do you
 think we would get the same proportion? Let's try sampling from the population
 again and see what happens. 
 
 ```{r 11-example-proportions4, echo = TRUE, message = FALSE, warning = FALSE}
-set.seed(2)
-samples_2 <- rep_sample_n(virtual_box, size = 40)
-choc_sample_2 <- summarize(samples_2,
-  n = sum(flavour == "chocolate"),
-  prop = sum(flavour == "chocolate") / 40
+set.seed(1234)
+sample_2 <- rep_sample_n(airbnb, size = 40)
+airbnb_sample_2 <- summarize(sample_2,
+  n = sum(room_type == "Entire home/apt"),
+  prop = sum(room_type == "Entire home/apt") / 40
 )
-choc_sample_2
+airbnb_sample_2
 ```
 
 Notice that we get a different value for our estimate this time. The
-proportion of chocolate Timbits in this sample is `r round(choc_sample_2$prop, 2)`. 
+proportion of entire home/apartment listings in this sample is `r round(airbnb_sample_2$prop, 3)`. 
 If we were to do this again, another random sample could also give a
 different result. Estimates vary from sample to sample 
 due to **sampling variability**. 
 
 But just how much should we expect the estimates of our random
-samples to vary? In order to understand this, we will simulate taking more samples
-of size 40 from our population of Timbits, and calculate the 
-proportion of chocolate Timbits in each sample. We can then
+samples to vary? To understand this, we will simulate taking more samples
+of size 40 from our population of listings and calculate the 
+proportion of entire home/apartment listings in each sample. We can then
 visualize the distribution of sample proportions we calculate. The distribution
 of the estimate for all possible samples of a given size (which we commonly refer to as $n$) from a population is
 called a **sampling distribution**. The sampling distribution will help us see
```

Comment:
should emphasize here that in the real world, since we dont have our pop, we can't actually construct the sampling distribution. but since in this eg for learning we do have it, we can actually visualize it directly.